controller_manager/test/test_controller_manager.cpp

```diff
@@ -486,7 +486,7 @@ TEST_P(TestControllerUpdateRates, check_the_controller_update_rate)
     time += rclcpp::Duration::from_seconds(0.01);
     if (update_counter % cm_update_rate == 0)
     {
-      const auto no_of_secs_passed = update_counter / cm_update_rate;
+      const double no_of_secs_passed = static_cast<double>(update_counter) / cm_update_rate;
```

Comment:
@destogl do we need this change?. In this particular case, the `update_counter` and `cm_update_rate` are of same type and from the previous `update_counter % cm_update_rate == 0`, they should result in a unsigned int.

Reply:
But do you want to divide two ints? was that the intention?

Reply:
yes, in order to retrieve the seconds passed. As per the loop above, it made sense to retrieve only that